### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Install" dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1371]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is pane.

**Narrator :**
Narrator announces as "Not on table" while using the table commands such as "Caps+F3", "Caps+F8". Also in scan mode, on using "Ctrl+alt+arrow" keys to navigate the table cells, narrator just announces as "Not on table".

**Expected Result:**
Control type should be Table.
Narrator table commands such as "Caps+F3", "Caps+F8" should work.

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![available software-table-inspect](https://user-images.githubusercontent.com/22956130/35680584-cf500d34-0780-11e8-8707-76ee4c3c1ff8.jpg)
![available software-table-narrator](https://user-images.githubusercontent.com/22956130/35680591-d2926cb2-0780-11e8-9279-09f18833bac2.jpg)

**Comment by Eskibear** [comment]
Duplicated with #1377 
It's the Eclipse IDE's dialog, out of the toolkit's scope.

As for the control type, already explained in #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749819
